addons/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/ModbusBindingConstants.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.modbus;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link ModbusBinding} class defines some constants
+ * public that might be used from other bundles as well.
+ *
+ * @author Sami Salonen - Initial contribution
+ * @author Nagy Attila Gabor - Split the original ModbusBindingConstants in two
+ */
+@NonNullByDefault
+public class ModbusBindingConstants {
```

Comment:
Should we give this class (or the original [ModbusBindingConstants](https://github.com/openhab/openhab2-addons/pull/4077/files#diff-0af7815991dc3c4419a8c33927a1783e) class) a different name to make it clear for everyone that one of them is internal and the other one is public API? Wdyt?

Reply:
Totally agree, I just could not come up with a good name. As I can see this separation is fairly new, I would like to ask you guys to propose a naming convention here.

Reply:
Off-the-cuff: `ModbusBindingConstants` and `ModbusBindingConstantsInternal`. :wink:

Reply:
Ok, renamed the class